assembl/static2/js/app/components/debate/multiColumns/columnsView.jsx

```diff
@@ -53,13 +64,30 @@ class ColumnsView extends React.Component {
       return countSynthesis >= 1;
     });
 
+    const { slug, phase, themeId } = routerParams;
+    const openShareSynthesisModal = (id) => {
+      const title = <Translate value="debate.shareSynthesis" />;
+      const url = `${window.location.protocol}//${window.location.host}${get('debate', {
```

Comment:
Try to use the react-router props provided from the highest level component to here. It's just a practice, not a hard-set rule.

Reply:
I mean that the router has `location` as well. You could perhaps pass that in as well.